enterprise/api/v1/views.py

```diff
@@ -239,7 +271,7 @@ class EnterpriseCustomerEntitlementViewSet(EnterpriseReadOnlyModelViewSet):
 
 class EnterpriseCustomerCatalogViewSet(EnterpriseReadOnlyModelViewSet):
     """
-    API Views for performing search through course discovery at the ``enterprise-catalogs`` API endpoint.
+    API Views for performing search through course discovery at the ``enterprise_catalogs`` API endpoint.
```

Comment:
I know it's just a comment and we'll do the change soon, but do you want to revert this too as per https://github.com/edx/edx-enterprise/pull/202#discussion_r142038744?

Reply:
No, paths will keep using underscores. API endpoint URL names will use dashes per DRF.

Reply:
Ah, I get it.